windfile.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import math
+import elev
+import datetime
+import bisect
+import time
+#from scipy import interpolate
+
+class WindFile:
+    def __init__(self, path):
+        self.data_npz = np.load("downtest2/" + path, "r")
+        self.data = self.data_npz['data']
+        self.time = self.data_npz['timestamp']
+        self.levels = self.data_npz['levels']
+        self.interval = self.data_npz['interval']
+        # TODO calculate resolution
```

Comment:
Upon second thought, this isn't that important so feel free to ignore for the moment